File: HHTXT/Selecter.vb
Public Class Selecter

    Private Sub Selecter_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        For Each i As Form In MDIParent1.MdiChildren
            ListBox1.Items.Add(i.Text)

        Next
    End Sub

    Private Sub ListBox1_SelectedIndexChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles ListBox1.SelectedIndexChanged
        Dim msg = MsgBox("Do you want to save " & ListBox1.SelectedItem & "?", MsgBoxStyle.YesNo)
        If msg = MsgBoxResult.Yes Then
            If SaveFileDialog1.ShowDialog() = Windows.Forms.DialogResult.OK Then
                Dim sw As IO.StreamWriter = New IO.StreamWriter(SaveFileDialog1.FileName)
                MsgBox("You are in a unused form of HHTXT, Please go to the main form")
                Application.Exit()
            End If
        End If
    End Sub
End Class

File: HHTXT/Selecter.cs
using System;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace WindowsApplication1
{
    public partial class Selecter
    {
        public Selecter()
        {
            InitializeComponent();
        }

        private void Selecter_Load(object sender, EventArgs e)
        {
            foreach (Form i in MDIParent1.instance.MdiChildren)

                ListBox1.Items.Add(i.Text);
        }

        private void ListBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
            var msg = Interaction.MsgBox(Operators.ConcatenateObject(Operators.ConcatenateObject("Do you want to save ", ListBox1.SelectedItem), "?"), MsgBoxStyle.YesNo);
            if (msg == MsgBoxResult.Yes)
            {
                if (SaveFileDialog1.ShowDialog() == DialogResult.OK)
                {
                    var sw = new System.IO.StreamWriter(SaveFileDialog1.FileName);
                    Interaction.MsgBox("You are in an unused form of HHTXT, please go to the main form");
                    Application.Exit();
                }
            }
        }
    }
}